features/exploredash/src/main/java/org/dash/wallet/features/exploredash/ui/SearchFragment.kt

```diff
@@ -84,7 +83,7 @@ class SearchFragment : Fragment(R.layout.fragment_search) {
     @Inject
     lateinit var analyticsService: AnalyticsService
     private val binding by viewBinding(FragmentSearchBinding::bind)
-    private val viewModel: ExploreViewModel by activityViewModels()
+    private val viewModel: ExploreViewModel by navGraphViewModels(R.id.explore_dash) { defaultViewModelProviderFactory }
```

Comment:
This also has to be changed everywhere where this view model is used, specifically in `ExploreMapFragment` and `FiltersDialog`. Right now, the location and filter functionality in Explore is broken.